File: Common/InMotionGIT.Common.Domain/Attributes/ElementSpanishDesigner.cs

namespace InMotionGIT.Common.Domain.Attributes;


/// <summary>
/// Define la etiqueta y tooltips en español asociado a la propiedad.
/// </summary>
/// <remarks></remarks>
public sealed class ElementSpanishDesigner : ElementDesignerAttribute
{

    public ElementSpanishDesigner(string caption) : base((int)Enumerations.EnumLanguage.Spanish, caption)
    {
    }

    public ElementSpanishDesigner(string caption, string toolTip) : base((int)Enumerations.EnumLanguage.Spanish, caption, toolTip)
    {
    }

}

File: Common/InMotionGIT.Common - Copy/MetadataCatalogAttributes/ElementSpanishDesigner.vb
Namespace Attributes

    ''' <summary>
    ''' Define la etiqueta y tooltips en español asociado a la propiedad.
    ''' </summary>
    ''' <remarks></remarks>
    Public NotInheritable Class ElementSpanishDesigner
        Inherits ElementDesignerAttribute

        Public Sub New(ByVal caption As String)
            MyBase.New(Enumerations.EnumLanguage.Spanish, caption)
        End Sub

        Public Sub New(ByVal caption As String, ByVal toolTip As String)
            MyBase.New(Enumerations.EnumLanguage.Spanish, caption, toolTip)
        End Sub

    End Class

End Namespace